defaults/main.yml

```diff
@@ -13,3 +13,7 @@ docker_version: "17.06"
 docker_default_config:
   storage-driver: devicemapper
   log-level: info
+
+docker_users: []
+# docker_users:
```

Comment:
I'd rather remove code as comments from the code itself as it confuses users. If this is an actual use case/example, maybe you can consider adding it to the README file.